src/extensions/mountainview/MountainView/ViewLauncher.tsx

```diff
@@ -1,11 +1,13 @@
 import React, { Fragment, FunctionComponent, useCallback } from 'react';
-import { Plugins, ViewPlugin } from '../../extensionInterface';
+import { Plugins, SortingSelection, SortingUnitViewPlugin, SortingViewPlugin } from '../../extensionInterface';
```

Comment:
Reorganizing and expanding the view launcher. This now provides a button for each SortingView plugin. Then, when units are selected, it provides a button for each selected unit for launching the MVSortingUnitView